src/Components/NavBar/__tests__/NavBar.test.tsx

```diff
@@ -11,24 +12,40 @@ jest.mock("Components/Search/SearchBar", () => {
     SearchBarQueryRenderer: () => <div />,
   }
 })
+jest.mock("Artsy/Analytics/useTracking")
+jest.mock("Artsy/Analytics/TrackingContext", () => {
+  return {
+    provideTracking: () => Component => Component,
+  }
+})
```

Comment:
So this all reminds me that in the order app we couldn't use the `mockTracking` helper at one point so we decided to leave `react-tracking` unmocked and we mocked out `postEvent` instead

https://github.com/artsy/reaction/blob/d562f3be73533115641130b299dc9395064d48b0/src/Apps/Order/Components/__tests__/PaymentPicker.test.tsx#L24-L28

And then in tests we were just like

https://github.com/artsy/reaction/blob/d562f3be73533115641130b299dc9395064d48b0/src/Apps/Order/Components/__tests__/PaymentPicker.test.tsx#L517-L522

Which I think is a lot simpler than mocking out `provideTracking` and `useTracking`. I wonder whether it makes sense to standardise on this?

Reply:
Oh interesting, like this will skip a couple steps. My first reactionis that it's a little bit of indirection that will be a bit hard to keep track of over time, like devs will naturally want to mock out the thing that's closest (vs `postEvent` which being lower level, is unseen to most). But it does seem simpler... will think about this for a bit and give it a spin.